apps/ai-backoffice/lib/workers/pipeline.ts

```
@@ -0,0 +1,126 @@
+}
+
+export type OutgoingEventData =
+  | ProgressEventData
+  | ReadyEventData
+  | ResultEventData
+
+class PipelineSingleton {
+  static instance?: Pipeline
+
+  static async init(...args: Parameters<typeof pipeline>) {
+    this.instance = await pipeline(...args)
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)